.github/workflows/deploy-moni-dev

```diff
@@ -0,0 +1,77 @@
+name: Deploy to ECR
+
+on:
+ 
+  push:
+    branches: [ moniafrica-master ]
```

Comment:
@hamidoke : is the branch is correct as we are using common repo and not moniafrica specific repo

Reply:
yes it's a common repo